docs/tutorials/03_quantum_kernel.ipynb

```diff
@@ -139,7 +189,15 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "With our training and testing datasets ready, we set up the `FidelityQuantumKernel` class to calculate a kernel matrix using the [ZZFeatureMap](https://qiskit.org/documentation/stubs/qiskit.circuit.library.ZZFeatureMap.html). We use the reference implementation of the `Sampler` primitive and the `ComputeUncompute` fidelity that computes overlaps between states. These are the default values and if you don't pass a `Sampler` or `Fidelity` instance, the same objects will be created automatically for you."
+    "With this our training and testing dataset is ready. Now, we set up the `FidelityQuantumKernel` ([link](https://github.com/Qiskit/qiskit-machine-learning/blob/main/qiskit_machine_learning/kernels/fidelity_quantum_kernel.py)) class to calculate a kernel matrix using the [ZZFeatureMap](https://qiskit.org/documentation/stubs/qiskit.circuit.library.ZZFeatureMap.html). \n",
```

Comment:
```suggestion
    "With this step, our training and testing dataset are ready. Now, we set up the [`FidelityQuantumKernel`](https://github.com/Qiskit/qiskit-machine-learning/blob/main/qiskit_machine_learning/kernels/fidelity_quantum_kernel.py) class to calculate a kernel matrix using the [ZZFeatureMap](https://qiskit.org/documentation/stubs/qiskit.circuit.library.ZZFeatureMap.html). \n",
```

Reply:
As a style preference, we have agreed with @adekusar-drl that it looks better to integrate the links into the text instead of writing "link", so you may have to change this in other tutorials too 😅.